packages/dds/sequence/src/test/intervalTestUtils.ts

```diff
@@ -129,8 +129,14 @@ async function assertPropertiesEqual(a: SharedString, b: SharedString): Promise<
 		for (const key of aKeys.concat(bKeys)) {
 			const aVal: unknown = aProps[key];
 			const bVal: unknown = bProps[key];
-			const aHandle = isObject(aVal) && isFluidHandle(aVal) ? await aVal.get() : aVal;
-			const bHandle = isObject(bVal) && isFluidHandle(bVal) ? await bVal.get() : bVal;
+			const aHandle =
+				isObject(aVal) && isFluidHandle(aVal)
+					? toFluidHandleInternal(aVal).absolutePath
+					: aVal;
+			const bHandle =
+				isObject(bVal) && isFluidHandle(bVal)
+					? toFluidHandleInternal(bVal).absolutePath
+					: bVal;
```

Comment:
I see that in main, it would be possible for aVal to be a handle pointing to non-handle bVal and have the two evaluate as equal.  And with this change it's possible for aVal to be a handle with absolutePath equal to a string bVal.  Maybe not realistic given the way the tests are written but would read easier and be more correct to test the handle case separately and explicitly.